### PR TITLE
Remove maketest.sh

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves  https://github.com/AdoptOpenJDK/openjdk-tests/issues/2235

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Testing: 

- z/OS : internal/Grinder/13859
- AIX: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/7050/
- Linux x64 : https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/7047/
- Windows: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/7049/

**Comment by Mesbah-Alam** [comment]
Local testing snapshot: 

```
Mesbahs-MacBook-Pro:test-removal-of-maketest.sh Mesbah_Alam@ca.ibm.com$ export TEST_JDK_HOME=/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/jre
Mesbahs-MacBook-Pro:test-removal-of-maketest.sh Mesbah_Alam@ca.ibm.com$ export BUILD_LIST=openjdk
Mesbahs-MacBook-Pro:test-removal-of-maketest.sh Mesbah_Alam@ca.ibm.com$ git clone https://github.com/Mesbah-Alam/openjdk-tests -b remove-maketest-sh
Cloning into 'openjdk-tests'...
remote: Enumerating objects: 7, done.
remote: Counting objects: 100% (7/7), done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 12109 (delta 0), reused 7 (delta 0), pack-reused 12102
Receiving objects: 100% (12109/12109), 63.23 MiB | 8.35 MiB/s, done.
Resolving deltas: 100% (7287/7287), done.
Updating files: 100% (1473/1473), done.
Mesbahs-MacBook-Pro:test-removal-of-maketest.sh Mesbah_Alam@ca.ibm.com$ cd openjdk-tests/
Mesbahs-MacBook-Pro:openjdk-tests Mesbah_Alam@ca.ibm.com$  TOP_DIR=$(pwd)
Mesbahs-MacBook-Pro:openjdk-tests Mesbah_Alam@ca.ibm.com$ TEST_DIR="$TOP_DIR"
Mesbahs-MacBook-Pro:openjdk-tests Mesbah_Alam@ca.ibm.com$ export BUILD_ROOT=$TOP_DIR/test-results
Mesbahs-MacBook-Pro:openjdk-tests Mesbah_Alam@ca.ibm.com$ ./get.sh -t $TEST_DIR
TESTDIR: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests
get testKitGen...
git clone -b master https://github.com/AdoptOpenJDK/TKG.git
check AdoptOpenJDK repo and TKG repo SHA
/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/getSHA.sh --repo_dir /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests --output_file /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
Check sha in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests and store the info in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
touch /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
================================================
timestamp: 20210217-114629
repo dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests
git repo: 
  Fetch URL: https://github.com/Mesbah-Alam/openjdk-tests
sha:
54d5c18ad747c4dc08210328a8ffd29ac312066b
/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/getSHA.sh --repo_dir /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG --output_file /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
Check sha in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG and store the info in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
================================================
timestamp: 20210217-114629
repo dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG
git repo: 
  Fetch URL: https://github.com/AdoptOpenJDK/TKG.git
sha:
ff55d88a720a49f5e69197cd83a8b87ad117c4db
get functional test material...
git clone --depth 1  https://github.com/eclipse/openj9.git
check OpenJ9 Repo sha
/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/getSHA.sh --repo_dir /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/openj9 --output_file /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
Check sha in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/openj9 and store the info in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/SHA.txt
================================================
timestamp: 20210217-114641
repo dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/openj9
git repo: 
  Fetch URL: https://github.com/eclipse/openj9.git
sha:
f1a0ca10ecbd6ca53c165e1723fea7da932fbf08
Mesbahs-MacBook-Pro:openjdk-tests Mesbah_Alam@ca.ibm.com$ cd ./TKG
Mesbahs-MacBook-Pro:TKG Mesbah_Alam@ca.ibm.com$ make compile
JAVA_HOME is set to /Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/jre
rm -f -r /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	mkdir -p /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	(ant -f ./scripts/build_tools.xml -DTEST_JDK_HOME=/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/jre -DTEST_ROOT=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/.. 2>&1; echo $? ) | tee /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log; \
	if [ -z $(tail -1 /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log | grep 0) ]; then perl scripts/moveDmp.pl --compileLogPath=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log --testRoot=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/..; false; else rm -f -r /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; fi
Buildfile: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/build_tools.xml

build:

clean:

init:
    [mkdir] Created dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin

getDependentLibs:
     [exec] /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib does not exist, creating one.
     [exec] mkdir /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] --------------------------------------------
     [exec] path is set to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] task is set to default
     [exec] dependencyList is set to json_simple
     [exec] --------------------------------------------
     [exec] Starting download third party dependent jars
     [exec] --------------------------------------------
     [exec] downloading dependent third party jars to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] downloading https://repo1.maven.org/maven2/com/googlecode/json-simple/json-simple/1.1.1/json-simple-1.1.1.jar
     [exec] --> file downloaded to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/json-simple.jar
     [exec] downloaded dependent third party jars successfully

compile:
     [echo] Ant version is Apache Ant(TM) version 1.10.5 compiled on July 10 2018
     [echo] ============COMPILER SETTINGS============
     [echo] ===fork:                         yes
     [echo] ===debug:                        on
    [javac] Compiling 23 source files to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin

BUILD FAILED
/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/build_tools.xml:58: The following error occurred while executing this line:
/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/build_tools.xml:37: Error running /Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/jre/bin/javac compiler

Total time: 2 seconds
1
make: *** [compileTools] Error 1
Mesbahs-MacBook-Pro:TKG Mesbah_Alam@ca.ibm.com$ export TEST_JDK_HOME=/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home
Mesbahs-MacBook-Pro:TKG Mesbah_Alam@ca.ibm.com$ make compile
JAVA_HOME is set to /Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home
rm -f -r /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	mkdir -p /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	(ant -f ./scripts/build_tools.xml -DTEST_JDK_HOME=/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home -DTEST_ROOT=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/.. 2>&1; echo $? ) | tee /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log; \
	if [ -z $(tail -1 /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log | grep 0) ]; then perl scripts/moveDmp.pl --compileLogPath=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log --testRoot=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/..; false; else rm -f -r /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; fi
Buildfile: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/build_tools.xml

build:

clean:
   [delete] Deleting directory /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin

init:
    [mkdir] Created dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin

getDependentLibs:
     [exec] --------------------------------------------
     [exec] path is set to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] task is set to default
     [exec] dependencyList is set to json_simple
     [exec] --------------------------------------------
     [exec] Starting download third party dependent jars
     [exec] --------------------------------------------
     [exec] downloading dependent third party jars to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/json-simple.jar exists with correct hash, not downloading
     [exec] downloaded dependent third party jars successfully

compile:
     [echo] Ant version is Apache Ant(TM) version 1.10.5 compiled on July 10 2018
     [echo] ============COMPILER SETTINGS============
     [echo] ===fork:                         yes
     [echo] ===debug:                        on
    [javac] Compiling 23 source files to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin

dist:
      [jar] Building jar: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/bin/TestKitGen.jar

BUILD SUCCESSFUL
Total time: 3 seconds
0
/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java -cp ./bin/TestKitGen.jar org.openj9.envInfo.EnvDetector
System.getProperty('java.vm.name')=Eclipse OpenJ9 VM

System.getProperty('java.vendor')=AdoptOpenJDK

System.getProperty('os.name')=Mac OS X

System.getProperty('os.arch')=x86_64

System.getProperty('java.fullversion')=1.8.0_282-b08
JRE 1.8.0 Mac OS X amd64-64-Bit Compressed References 20210120_839 (JIT enabled, AOT enabled)
OpenJ9   - 345e1b09e
OMR      - 741e94ea8
JCL      - ab07c6a8fd based on jdk8u282-b08

System.getProperty('sun.arch.data.model')=64

****************************** MACHINE INFO ******************************
uname : Darwin Mesbahs-MacBook-Pro.local 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64
cpuCores : 
	cat: /proc/cpuinfo: No such file or directory
	       0
sysArch : x86_64
procArch : i386
sysOS : Darwin
ulimit : 
	core file size          (blocks, -c) 0
	data seg size           (kbytes, -d) unlimited
	file size               (blocks, -f) unlimited
	max locked memory       (kbytes, -l) unlimited
	max memory size         (kbytes, -m) unlimited
	open files                      (-n) 24576
	pipe size            (512 bytes, -p) 1
	stack size              (kbytes, -s) 65532
	cpu time               (seconds, -t) unlimited
	max user processes              (-u) 2784
	virtual memory          (kbytes, -v) unlimited
antVersion : Apache Ant(TM) version 1.10.5 compiled on July 10 2018
makeVersion : 
	GNU Make 3.81
	Copyright (C) 2006  Free Software Foundation, Inc.
	This is free software; see the source for copying conditions.
	There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
	PARTICULAR PURPOSE.
	
	This program built for i386-apple-darwin11.3.0
perlVersion : 
	
	This is perl 5, version 28, subversion 2 (v5.28.2) built for darwin-thread-multi-2level
	(with 2 registered patches, see perl -V for more detail)
	
	Copyright 1987-2019, Larry Wall
	
	Perl may be copied only under the terms of either the Artistic License or the
	GNU General Public License, which may be found in the Perl 5 source kit.
	
	Complete documentation for Perl, including FAQ lists, should be found on
	this system using "man perl" or "perldoc perl".  If you have access to the
	Internet, point your browser at http://www.perl.org/, the Perl Home Page.
	
curlVersion : 
	curl 7.64.1 (x86_64-apple-darwin20.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.41.0
	Release-Date: 2019-03-27
	Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp smb smbs smtp smtps telnet tftp 
	Features: AsynchDNS GSS-API HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz MultiSSL NTLM NTLM_WB SPNEGO SSL UnixSockets
vmVendor : Oracle Corporation
vmVersion : openj9-0.24.0
Total memory (bytes) : 8388608
Free memory (bytes) : 6496944
File path : /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG
Total space (bytes) : 499963170816
Free space (bytes) : 266900885504
Usable space (bytes) : 248598384640
**************************************************************************

/Library/Developer/CommandLineTools/usr/bin/make -f compile.mk compile
rm -f -r /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	mkdir -p /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation; \
	(ant -f scripts/build_test.xml -DTEST_ROOT=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/.. -DBUILD_ROOT=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results -DJDK_VERSION=8 -DJDK_IMPL=openj9 -DJCL_VERSION=latest -DBUILD_LIST=openjdk -DRESOURCES_DIR=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/TestConfig/resources -DSPEC=osx_x86-64_cmprssptrs -DTEST_JDK_HOME=/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home -DJVM_VERSION=openjdk8-openj9 -DLIB_DIR=/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib 2>&1; echo $? ) | tee "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log"; \
	if [ -z "$(tail -1 /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log | grep 0)" ]; then perl scripts/moveDmp.pl --compileLogPath="/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation/compilation.log" --testRoot="/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/.."; false; else rm -f -r "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_compilation"; fi
Buildfile: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/scripts/build_test.xml
     [echo]  build.list is TestConfig/build.xml,openjdk/build.xml 

-create_test_directory:
    [mkdir] Created dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results

stage_test_material:
     [copy] Copying 87 files to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results

build:

init:
    [mkdir] Created dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/TestConfig

build:
     [copy] Copying 9 files to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/TestConfig
     [copy] Copied 4 empty directories to 2 empty directories under /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/TestConfig
     [copy] Copying 1 file to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/TestConfig

build:

getDependentLibsCheck:

getDependentLibs:
     [exec] --------------------------------------------
     [exec] path is set to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] task is set to default
     [exec] dependencyList is set to tohandler_simple
     [exec] --------------------------------------------
     [exec] Starting download third party dependent jars
     [exec] --------------------------------------------
     [exec] downloading dependent third party jars to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib
     [exec] downloading https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/lastSuccessfulBuild/artifact/openj9jtregtimeouthandler.jar.sha256sum.txt
     [exec] --> file downloaded to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/openj9jtregtimeouthandler.jar.sha256sum.txt
     [exec] downloading https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/lastSuccessfulBuild/artifact/openj9jtregtimeouthandler.jar
     [exec] --> file downloaded to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/openj9jtregtimeouthandler.jar
     [exec] downloading https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/lastSuccessfulBuild/artifact/openj9jtregtimeouthandler.jar.sha256sum.txt
     [exec] --> file downloaded to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/openj9jtregtimeouthandler.jar.sha256sum.txt
     [exec] downloaded dependent third party jars successfully

getJtreg:
    [mkdir] Created dir: /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk
     [echo] Getting Jtreg from https://ci.adoptopenjdk.net/job/jtreg/lastSuccessfulBuild/artifact/jtreg-4.2.0-tip.tar.gz

openjdk-jdk.check:

getOpenjdk:
     [echo] git clone --depth 1 -q  https://github.com/ibmruntimes/openj9-openjdk-jdk8.git
     [exec] Check sha in openjdk-jdk and store the info in /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/SHA.txt
     [exec] ================================================
     [exec] timestamp: 20210217-114904
     [exec] repo dir: openjdk-jdk
     [exec] git repo: 
     [exec]   Fetch URL: https://github.com/ibmruntimes/openj9-openjdk-jdk8.git
     [exec] sha:
     [exec] 77e609e87b99431c222bf7c3f0c81a2d713bcd77

dist:
     [copy] Copying 24 files to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk

BUILD SUCCESSFUL
Total time: 54 seconds
0
Mesbahs-MacBook-Pro:TKG Mesbah_Alam@ca.ibm.com$ make _sanity.openjdk
JAVA_HOME is set to /Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home
/Library/Developer/CommandLineTools/usr/bin/make -f makeGen.mk AUTO_DETECT= MODE=tests TESTTARGET=sanity.openjdk TESTLIST=
perl scripts/configure.pl
/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java -cp "./bin/TestKitGen.jar:./lib/json-simple.jar" org.testKitGen.MainRunner --mode=tests --spec=osx_x86-64_cmprssptrs --jdkVersion=8 --impl=openj9 --vendor="adoptopenjdk" --buildList=openjdk --iterations=1 --testFlag= --testTarget=sanity.openjdk --testList= --numOfMachines= --testTime= --TRSSURL= 
Modes data parsed from resources/modes.xml and resources/ottawa.csv.

Starting to generate test make files.

Parsing /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../openjdk/playlist.xml

Generated /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../openjdk/autoGen.mk

Generated /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../autoGen.mk

Generated /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/utils.mk

Make files are generated successfully.

/Library/Developer/CommandLineTools/usr/bin/make -f runtest.mk _sanity.openjdk

Running make 3.81
set TEST_ROOT to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/..
set JDK_VERSION to 8
set JDK_IMPL to openj9
set JVM_VERSION to openjdk8-openj9
set JCL_VERSION to latest
set JAVA_HOME to /Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home
set SPEC to osx_x86-64_cmprssptrs
Running sanity.openjdk ...
There are 9 test targets in sanity.openjdk
"/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java" -version
openjdk version "1.8.0_282"
OpenJDK Runtime Environment (build 1.8.0_282-b08)
Eclipse OpenJ9 VM (build openj9-0.24.0, JRE 1.8.0 Mac OS X amd64-64-Bit Compressed References 20210120_839 (JIT enabled, AOT enabled)
OpenJ9   - 345e1b09e
OMR      - 741e94ea8
JCL      - ab07c6a8fd based on jdk8u282-b08)

===============================================
Running test jdk_lang_0 ...
===============================================
jdk_lang_0 Start Time: Wed Feb 17 11:49:11 2021 Epoch Time (ms): 1613580551444
"/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java" -Xshareclasses:destroyAll; "/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";

Attempting to destroy all caches in cacheDir /Users/Mesbah_Alam@ca.ibm.com/javasharedresources/

JVMSHRC806I Compressed references persistent shared cache "sharedcc_Mesbah_Alam@ca.ibm.com" has been destroyed. Use option -Xnocompressedrefs if you want to destroy a non-compressed references cache.
JVMSHRC005I No shared class caches available
cache cleanup done
variation: -Xdump:system:none -Xdump:heap:none -Xdump:system:events=gpf+abort+traceassert+corruptcache -XX:-JITServerTechPreviewMessage Mode150
JVM_OPTIONS:  -Xdump:system:none -Xdump:heap:none -Xdump:system:events=gpf+abort+traceassert+corruptcache -XX:-JITServerTechPreviewMessage -XX:+UseCompressedOops 
{ itercnt=1; \
	mkdir -p "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_16135805508292/jdk_lang_0"; \
	cd "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_16135805508292/jdk_lang_0"; \
	"/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home/bin/java" -Xmx512m -jar "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk/jtreg/lib/jtreg.jar" \
	-agentvm -a -ea -esa -v:fail,error,time,nopass -retain:fail,error,*.dmp,javacore.*,heapdump.*,*.trc -ignore:quiet -timeoutFactor:8 -xml:verify -concurrency:4  -vmoptions:"-Xmx512m  -Xdump:system:none -Xdump:heap:none -Xdump:system:events=gpf+abort+traceassert+corruptcache -XX:-JITServerTechPreviewMessage -XX:+UseCompressedOops " \
	-timeoutHandler:jtreg.openj9.CoreDumpTimeoutHandler -timeoutHandlerDir:"/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/lib/openj9jtregtimeouthandler.jar" \
	-w ""/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_16135805508292/jdk_lang_0"/work" \
	-r "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk/report" \
	-jdk:"/Users/Mesbah_Alam@ca.ibm.com/local_sdk/openj9-jdk8/jdk8u282-b08/Contents/Home" \
	-exclude:"/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../openjdk/openjdk-jdk/jdk/test/ProblemList.txt" \
	-exclude:"/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk/ProblemList_openjdk8-openj9.txt" \
	"/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../openjdk/openjdk-jdk/jdk/test:jdk_lang"; \
	if [ $? -eq 0 ] ; then echo ""; echo "jdk_lang_0""_PASSED"; echo ""; cd /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/..;  else echo ""; echo "jdk_lang_0""_FAILED"; echo ""; fi; } 2>&1 | tee -a "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_16135805508292/TestTargetResult";
Directory "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/../TKG/output_16135805508292/jdk_lang_0/work" not found: creating
Directory "/Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk/report" not found: creating
XML output with verification to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/output_16135805508292/jdk_lang_0/work




Test results: passed: 436
Report written to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/test-results/openjdk/report/html/report.html
Results written to /Users/Mesbah_Alam@ca.ibm.com/test-removal-of-maketest.sh/openjdk-tests/TKG/output_16135805508292/jdk_lang_0/work

jdk_lang_0_PASSED

<<<<<<<Truncated output>>>>> 
```

**Comment by smlambert** [comment]
Adding a note for discussion... 

We have: 
- makeTest(testParam) which is the method that executes a test
- make_test() which is the method that calls make compile (to build the test material)

While its not directly related to this PR, suggest renaming make_test() to be something like makeCompileTest() or whatever naming convention we seem to be following in this file (seem to be 2 competing styles, with camelCase naming convention being most prevalent), to make it more clear its different from makeTest.

@Mesbah-Alam - Can you also check the doc directory to make sure there are no references to maketest.sh?

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -474,13 +476,14 @@ def get_sources() {
 }
 
 def make_test() {
+	String makeTestCmd = "if [ `uname` = AIX ] || [ `uname` = SunOS ] || [ `uname` = *BSD ]; then MAKE=gmake; else MAKE=make; fi; cd ./openjdk-tests/TKG; \$MAKE compile"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:479]: Instead of having this defined multiple times, can we have one variable for it?

**Comment by Mesbah-Alam** [comment]
> Can you also check the doc directory to make sure there are no references to maketest.sh?

- There were two references to `maketest.sh` in documentation. They have been updated in this PR. 

> renaming make_test() to be something like makeCompileTest() 

We could do this update in this PR! 

@ShelleyLambert

File: buildenv/jenkins/JenkinsfileBase (hunk 1 above)
Comment (Mesbah-Alam) [reply buildenv/jenkins/JenkinsfileBase]: Hi @llxia , 

Yes, I've updated the PR based on your suggestion. We now have it as a variable. 

Retested: 
Internal/Grinder/13872
Internal/Grinder/13874

**Review by smlambert — APPROVED** [review]
thanks @Mesbah-Alam - I am very pleased that we as a community continue to strip down, simplify and refine what we do!

**Review by sophia-guo — APPROVED** [review]
LGTM

**Pull request merged**